pkg/systemd/terminal.jsx

```diff
@@ -220,86 +220,90 @@ const _ = cockpit.gettext;
                             channel={this.state.channel}
                             theme={this.state.theme}
                             fontSize={this.state.size}
-                            parentId="the-terminal"
+                    parentId="cockpit-terminal"
```

Comment:
Whitespace! :)

Reply:
We need better formatter usage :pensive: